pkg/syncer/specsyncer.go

```diff
@@ -34,7 +37,7 @@ import (
 	"k8s.io/klog/v2"
 	"k8s.io/utils/pointer"
 
-	nscontroller "github.com/kcp-dev/kcp/pkg/reconciler/workload/namespace"
+	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
 )
 
 func deepEqualApartFromStatus(oldObj, newObj interface{}) bool {
```

Comment:
make this strongly typed with unstructured. Where this is called you know everything is unstructured.

Reply:
done